backend/src/attraction/services/attraction_service.py

```diff
@@ -18,11 +18,11 @@
 
 class AttractionService:
     def __init__(
-        self,
-        repository: Repository,
-        cache: ICacheStorage,
-        cloud_storage: ICloudStorage,
-        geo_service: MapsCoService,
+            self,
+            repository: Repository,
+            cache: ICacheStorage,
+            cloud_storage: ICloudStorage,
+            geo_service: MapsCoService,
```

Comment:
This is wrong. We should type it as interface. The reason why its wrong is that you are importing MapsCoService to use it as a type, because of it there is still hard coupling between `MapsCoService` and `AttractionService`